edivorce/apps/core/templates/question/01_orders.html

```diff
@@ -230,17 +238,13 @@ <h1>
                     </div>
                 </div>
                 <div class="collapse" id="collapseOrders">
-                    <ol>
-                        <li>
-                            If your Separation Agreement is 'filed' at a Court Registry and you do not wish it changed, you may choose NOT to select this option. If you do wish to change what was agreed upon in your Separation Agreement, you must go to the Registry where it is filed.
-                        </li>
-                        <li>
-                            If you already have a Court Order pertaining to Children, you must go to the Court Registry where it is filed to apply to change or cancel terms.
-                        </li>
-                    </ol>
+                    <p>
+                        If your Separation Agreement is 'filed' at a Court Registry and you do not wish it changed, you may choose NOT to select this option. If you do wish to change what was agreed upon in your Separation Agreement, you must go to the Registry where it is filed.
+
```

Comment:
@dominikszopa Can you wrap each paragraph with its own `<p>` tag. Paragraph tag won't kept the format and this will be just one giant paragraph rather than two separate paragraphs.

Reply:
Good catch, added in the latest commit.